examples/similarity_search/add_faces.py

```diff
@@ -1,23 +1,35 @@
 import argparse
 from aperturedb import Utils, ParallelLoader
-from facenet import generate_embedding
 import dbinfo
 from CelebADataKaggle import CelebADataKaggle
+from aperturedb.transformers.facenet_pytorch_embeddings import FacenetPytorchEmbeddings
+from aperturedb.transformers.common_properties import CommonProperties
+from aperturedb.transformers.image_properties import ImageProperties
 
 search_set_name = "similar_celebreties"
 
 
 def main(params):
     utils = Utils.Utils(dbinfo.create_connector())
     utils.remove_descriptorset(search_set_name)
-    utils.add_descriptorset(search_set_name, 512,
-                            metric="L2", engine="FaissFlat")
-
-    dataset = CelebADataKaggle(
-        records_count=params.images_count,
-        embedding_generator=generate_embedding,
-        search_set_name=search_set_name
-    )
+
+    dataset = CelebADataKaggle()
+
+    # Here's a pipeline that adds extra properties to the celebA dataset
+    dataset = CommonProperties(
+        dataset,
+        adb_data_source="celebA",
+        adb_main_object="Face",
+        adb_annoted=True)
+
+    # some useful properties for the images
+    dataset = ImageProperties(dataset)
+
+    # Add the embeddings generated through facenet.
+    dataset = FacenetPytorchEmbeddings(dataset)
```

Comment:
Is this example now duplicated here and in our Python package? Which one does docs use?

Reply:
The difference in this and our adb ingest is the FacenetEmbeddings. docs will not show FacenetEmbeddings as an option, this although they will callout that such a class exists if anyone wants to 

- use it, as it is.
- refer to it to build something custom.